src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/RoboticsCallbackHandler.java

```diff
@@ -72,9 +74,10 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
     }
 
     private boolean checkCaseValidToSendToRobotics(Callback<SscsCaseData> callback) {
-        if (readyToListFeatureEnabled && callback.getEvent() != RESEND_CASE_TO_GAPS2 && callback.getCaseDetails().getCaseData().getAppeal().getBenefitType().getCode().equalsIgnoreCase("pip")) {
+        if (readyToListFeatureEnabled && callback.getEvent() != RESEND_CASE_TO_GAPS2
```

Comment:
I think you can just remove the feature flag here. The feature will be controlled from Tribunals (and Bulk Scan maybe?) as and when we expand the list of offices that set the createdInGapsFrom field to ready to list. At the moment the list of offices is empty so feature is effectively off.